README.md

```diff
@@ -60,6 +60,51 @@ This option enables you to develop your custom MATLAB commands for running tests
 
   ![new_custom_script_example](https://user-images.githubusercontent.com/47204011/55625021-32904400-57c5-11e9-86b7-478b930796c0.png)
 
+## Configuring "Multi-configuration"(matrix) project.
+Jenkins plugin for MATLAB can be configured to run "Multi-configuration" (matrix) projects. Which allows user to run the same build steps with different configurations, such as testing on multiple environments or multiple MATLAB versions installed on the system.
```

Comment:
I think it is more accurate to say that the Jenkins plugin supports running inside of multi-configuration jobs, since it is just a possible step in the job.  Perhaps something like:   

```
The Jenkins plugin for MATLAB can be used inside of "multi-configuration", or matrix, projects.  Matrix projects allow jobs to be repeated with different configurations. Matrix projects could be used for testing on multiple platforms or testing against multiple MATLAB versions.
```

Reply:
Suggestion with slight change: 

The Jenkins plugin for MATLAB can be used inside of "Multi-configuration" (matrix) projects.  Matrix projects allow jobs to be repeated with different configurations, such as testing on multiple platforms or testing against multiple MATLAB versions.